motionplan/motionPlanner.go

```diff
@@ -103,7 +104,35 @@ func (req *PlanRequest) validatePlanRequest() error {
 		return errors.New("PlanRequest must have at least one goal")
 	}
 
-	// Validate the goals. Each goal with a pose must not alos have a configuration specified. The parent frame of the pose must exist.
+	if useOc, ok := req.Options["meshes_as_octrees"].(bool); useOc && ok {
```

Comment:
should `"meshes_as_octrees"` go into the `plannerOptions` struct?

Reply:
I just looked through the code and the whole plannerOptions struct is in a really sorry state at the moment... Going to think about how best to do this

Reply:
> the whole plannerOptions struct is in a really sorry state at the moment

two options:
1. keep as is
2. put into plannerOptions

regardless we should make a ticket to get it out of the "sorry state" and have that be a separate body of work.

Reply:
ticketed.  Keeping as is because this is not actually possible to put in that struct without a lot of effort